packages/utilities/aria-utils/src/collections/section.ts

```diff
@@ -6,5 +6,8 @@ import {HTMLNextUIProps, As} from "@nextui-org/system";
  * A modified version of the SectionProps from @react-types/shared, with the addition of the NextUI props.
  *
  */
-export type SectionProps<Type extends As = "div", T extends object = {}> = BaseSectionProps<T> &
-  Omit<HTMLNextUIProps<Type>, "children">;
+export type SectionProps<
+  Type extends As = "div",
+  T extends object = {},
+  OmitKeys extends string = "children" | "title",
+> = BaseSectionProps<T> & Omit<HTMLNextUIProps<Type>, OmitKeys>;
```

Comment:
```suggestion
export type SectionProps<
  Type extends As = "div",
  T extends object = {},
  OmitKeys extends string = "children" | "title",
> = BaseSectionProps<T> & Omit<HTMLNextUIProps<Type>, OmitKeys>;
```
```suggestion
export type SectionProps<Type extends As = "div", T extends object = {}> = BaseSectionProps<T> &
  Omit<HTMLNextUIProps<Type>, "children" | "title">;
```

It would be better if the `title` of `BaseSectionProps` is used.
https://github.com/adobe/react-spectrum/blob/07431f4b1809bc2b1b015149737e08db60d5c332/packages/%40react-types/shared/src/collections.d.ts#L44-L45